.github/workflows/mise.yml

```
@@ -0,0 +1,48 @@
+          cache: true
+          experimental: true 
+      - name: Setup database.yml
+        run: |
+            cp config/database.example.yml config/database.yml
+      - name: Install Ruby dependencies
+        run: |
+            gem install bundler
+            bundle install
+      - name: Run Rails test
+        run: |
+            bin/rails runner 'puts "Hello CircuitVerse!"'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)